python/modules/DeepTopProducer.py

```diff
@@ -349,6 +345,30 @@ def calculateResTopSFWeight(self, res):
             numerator_up = ((1 + uncert_up_tagged)*resTopSF_tagged*resTopEff_tagged).prod() * (1 - ((1 + uncert_up_notTagged)*resTopSF_notTagged*resTopEff_notTagged)).prod()
             numerator_dn = ((1 - uncert_dn_tagged)*resTopSF_tagged*resTopEff_tagged).prod() * (1 - ((1 - uncert_dn_notTagged)*resTopSF_notTagged*resTopEff_notTagged)).prod()
 
+            # check if resolved top closure uncertainty is needed and apply it if necessary
+            if ((resTopGM == 0) & discCut).any():
```

Comment:
@pastika I guess this check isn't needed  and I should revert to the master code?

Reply:
no, this IS the closure uncertainty

Reply:
Ok, I will merge this file from master manually. Will it impact the cross check study Anna and Caleb did?

Reply:
Hold on, this part of code is in this branch, while no on master or PR117. I am confused